### ISSUE TITLE
Mark GTMProbe as synthetic

### ISSUE THREAD

**Issue by SergeyKanzhelev** [Microsoft/ApplicationInsights-dotnet-server#180]
User agent containing `GTMProbe` should be marked as synthetic traffic coming from the Azure Traffic Manager

**Comment by d-saravanan** [comment]
Small clarification, can you please confirm if we need to add another const string for "GTMProbe" user agent in the file `SyntheticUserAgentTelemetryInitializer` and then execute the test cases in `SyntheticUserAgentTelemetryInitializerTest`.

**Comment by SergeyKanzhelev** [comment]
Yes, this is pretty much the scope of the work here. It is important though to try it out on live environment. @d-saravanan are you considering contributing a Pull Request?

**Comment by d-saravanan** [comment]
@SergeyKanzhelev Thank you for the response. I was actually trying to contribute, but I am afraid on how I can test in Azure behind traffic managers in a live environment, I don't have applications with load balancer and traffic managers for testing.

**Comment by SergeyKanzhelev** [comment]
@d-saravanan what kind of issue are you looking for to contribute? I looked and we didn't really done a great job marking issues with the "help wanted" tag. I can help you navigate issues and suggest something you'd like to try.

For this specific issue - exact reason it was hanging around for so long is that team haven't formalized their User Agent String: https://feedback.azure.com/forums/217313-networking/suggestions/6486357-formalize-the-traffic-manager-user-agent-string In fact they recently changed it. That's why some testing was needed. If you want to learn how to configure Traffic Manager - you can continue with this issues. Otherwise - let me know so I can help you find something else.

**Comment by d-saravanan** [comment]
Thanks a lot @SergeyKanzhelev 
Sorry for the delay in responding back. I would like to request you to help me by giving some issues that I can work with and contribute. I do not have enterprise Azure accounts, however, I can perform changes and I can use free account to test and submit pull-request. Kindly suggest / help me.
